src/main/target/SPRACINGF4EVO/target.c

```diff
@@ -0,0 +1,64 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdint.h>
+
+#include <platform.h>
+#include "drivers/io.h"
+
+#include "drivers/timer.h"
+#include "drivers/timer_def.h"
+#include "drivers/dma.h"
+
+
+const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
+    { TIM9,  IO_TAG(PA3),   TIM_Channel_2, TIM_USE_PPM | TIM_USE_PWM,   0, GPIO_AF_TIM9, NULL, 0, 0 }, // PPM / PWM1 / UART2 RX
```

Comment:
@hydra I've a problema with the ESC 2 definition:
```
{ TIM8,  IO_TAG(PC7),  TIM_Channel_2, TIM_USE_MOTOR,                1, GPIO_AF_TIM8,  DMA2_Stream3, DMA_Channel_7, DMA2_ST3_HANDLER }, // ESC 2
```
When changing it to `DEF_TIM`form:
```
DEF_TIM(TIM8,  CH2, PC7,  TIM_Channel_2, TIM_USE_MOTOR,               1, X), // ESC 2
```
If I put a 0 in the X position, then it will be `DMA_Channel_0` and if I put a 1 in the X position then it will be `DMA2_ST2`. Both are incorrect. Sorry if it's a newbie question, but I don't know if I must add a 2 to the definitions (modifying the `timer_def.h`) or something is wrong.

It's the only one definition with problems. The others case perfectly.